### PR TITLE
Add tests for iffcmp??u / ifdcmp??u

### PR DESCRIPTION
**Pull request by janvrany**

### CODE REVIEW

**Comment by janvrany** [comment]
@genie-omr build all

File: fvtest/compilertriltest/CompareTest.cpp [hunk 1]
```diff
@@ -1422,3 +1422,249 @@ INSTANTIATE_TEST_CASE_P(CompareTest, DoubleCompareOrUnordered, ::testing::Combin
...
+
+TEST_P(FloatIfCompareOrUnordered, UsingConst) {
+    SKIP_ON_RISCV(MissingImplementation);
+    SKIP_ON_HAMMER(KnownBug) << "The x86_64 code generator crashes (see issue #5122)";
+
+    auto param = TRTest::to_struct(GetParam());
+
+    if ( param.opcode == "iffcmpneu" && (std::isnan(param.lhs) || std::isnan(param.rhs)) ) {
+        SKIP_ON_POWER(KnownBug) << "iffcmpneu returns wrong value on POWER (see #5152)";
+    }
```
Comment (aviansie-ben) [inline fvtest/compilertriltest/CompareTest.cpp:1472]: Only the `ne` variants should be affected by #5152 not the `neu` variants, so these checks should be removed.

**Comment by janvrany** [comment]
@genie-omr build plinux, aix

**Comment by 0xdaryl** [comment]
@genie-omr build all

**Comment by 0xdaryl** [comment]
Thanks for the tests @janvrany !  Great contribution!

**Pull request merged**